### DISCUSSION TITLE
PretendardJP font(woff, woff2) 소스 내에 삽입하여 사용시 사용불가

### DISCUSSION THREAD

**Discussion by qkrdkwl9090** [orioncactus/pretendard#76] [Q&A] [answered]
pretendard-Bold.woff 등 폰트를 사용하다, 홈페이지에 있는 일본어 버전 폰트 다운받아 pretendardJP-Bold.woff로 폰트 변경하였는데,
대체 폰트로 대체됩니다. woff2로 수정 후, 적용해도 똑같네요.

woff파일을 다운받아 소스에 직접 넣는것 말고 추가로 해야하는것이 있을까요?

font-family: 'Pretendard JP', ...

font-face {
font-family: 'Pretendard JP';
font-weight: 700;
src: (생략)/static/fonts/PretendardJP-Bold.woff')}') format('woff');
}

**Comment by orioncactus** [comment, marked as answer]
안녕하세요!

코드 입력이 잘못된 것 같아요. 적어주신 것처럼 `src: (생략)/static/fonts/PretendardJP-Bold.woff')}') format('woff');`로 적으신 게 맞을까요? 중간에 `)}')`때문에 오류가 생기는 듯해서, `src: (생략)/static/fonts/PretendardJP-Bold.woff') format('woff');`와 같이 해보시고 안 된다면 알려주세요!

**Reply by qkrdkwl9090** [reply]
@orioncactus 생략된 부분 때문에 남은거라 맞습니다.

**Reply by orioncactus** [reply]
🤔 Safari에서는 정상적으로 표시되는데 Chrome에서만 말씀하신 것처럼 표시되지 않네요. 한 번 확인해보겠습니다. 그동안 Pretendard JP를 웹폰트로 정상적으로 사용하시려면 Pretendard JP 다이나믹 서브셋을 사용하시는 것을 추천드립니다.

#### HTML
```html
<link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard-jp-dynamic-subset.css" />
```

#### CSS
```css
@import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard-jp-dynamic-subset.css');
```

**Reply by orioncactus** [reply]
이 문제는 이슈가 맞아 이슈에서 계속하겠습니다!
- #75
